### PR TITLE
Convert youtube links to embedded videos in cheep

### LINKED ISSUE #229: As a developer I would like to to convert youtube url links (after they've been posted in a cheep) to iframes (extra feature)
Now is the time to try more things, and this should be possible to develop fairly easy.

Acceptance criteria:

- [x] youtube links are converted to youtube embeds when the user has posted a cheep

### PR DESCRIPTION
**Pull request by JonasSkjodt**
Nobody asked for this but here it is. An exciting feature where Chirp! users are now able to paste a youtube link in their cheeps, where the program then converts it into an embedded youtube video.

Resolves #229

Edit: this is for reference only and shouldnt be merged yet

### CODE REVIEW

**Review by Slug-Boi — APPROVED** [review]
I didn't even think to do this! But its super duper cool and looks supprisingly easy to do using the IFrame stuff. Really good job. 100% Approved for merge! =)

**Review by Haruyzal — COMMENTED** [review]
Please also note a bug I found while playing around in this branch: #232

**Review by August-Brandt — COMMENTED** [review]
When the comments are addressed this looks good for merge

File: src/Chirp.Web/Pages/UserProfile.cshtml.cs [hunk 1]
```diff
@@ -38,14 +43,20 @@ public async Task<ActionResult> OnGetAsync()
 
     public async Task<IActionResult> OnPostForgetMeAsync()
     {
-        if(!User.Identity.IsAuthenticated) {
+        if(User?.Identity?.IsAuthenticated == false) {
```
[the remark this answers is not in the document]
Comment (JonasSkjodt) [reply src/Chirp.Web/Pages/UserProfile.cshtml.cs:46]: converted to try-catch

**Pull request closed without merging**